### PR TITLE
Carousel dummy data

### PR DESCRIPTION
**Pull request by milofultz**
Making the dummy data work with the Carousel component. Gave me a chance to refactor it down to state also.

### CODE REVIEW

**Comment by milofultz** [comment]
Most of the changes were me being anal and fixing the layout of the dummy data hahaha

File: client/src/components/RelatedItemsAndOutfit.jsx [hunk 1]
```diff
@@ -1,21 +1,50 @@
 import React from 'react';
 import Carousel from './Carousel.jsx';
+import { productsMock, productInfoMock, productStylesMock, relatedProductsMock } from '../mockData/productInfoMock.js';
+import { reviewsMock, reviewsMetaMock } from '../mockData/reviewsMock.js';
```
Comment (baudot) [inline client/src/components/RelatedItemsAndOutfit.jsx:4]: Not passing data down from the top?

I thought that's how we were handling state, and this isn't something I'd hold up the PR over, but I figured if it were me, I'd have gone ahead and modeled passing the data down from ProductDetailPage.

**Review by baudot — APPROVED** [review]
Looks good. I'd have gone ahead and passed the data where you're using it, while doing the other rewrites, rather than loading the mocks locally. But that's nothing worth holding the PR up over.

File: client/src/components/RelatedItemsAndOutfit.jsx (hunk 1 above)
Comment (milofultz) [reply client/src/components/RelatedItemsAndOutfit.jsx:4]: you're right, just didn't want to get in the way of other people's stuff. Keep my mess within my area until we can coordinate our big picture otp-level state

Comment (milofultz) [reply client/src/components/RelatedItemsAndOutfit.jsx:4]: But that is the better way to do it hahaha

**Pull request merged**